README.md

```diff
@@ -4,76 +4,122 @@
 
-This extension assumes you have the following binaries on your *PATH*:
+**Procedure**
 
-- `mvn` (for analyzing Java applications)
-- `npm` (for analyzing Node applications)
+1. Open the Visual Studio Code application.
+2. Click the **Extensions** icon from the **Activity Bar** or click **View** from the file menu, and click **Extensions**.
+3. Find the installed **Dependency Analytics** extension, and click on the **Gear** icon.
+4. Click **Extension Settings**.
 
-**Note:** By default, the `mvn/npm` command is executed directly in the terminal, which requires that `mvn/npm` is found in your system environment `PATH`.
-In case it is located elsewhere you can configure this via preferences in VS Code. To do this access the [extension settings](https://code.visualstudio.com/docs/getstarted/settings) within your VS code instance, switch to *Workspace* tab, search for *\<Ecosystem\> Executable Path* and edit the *Path* setting.
-You can also the `settings.json` file and add the following:
+   ![Red Hat Dependency Analytics extension workspace settings](images/screenshots/extension-workspace-settings.png)
 
-```
-{
-    ...
-    "maven.executable.path": "/path-to-maven-home/bin/mvn"
-    "npm.executable.path": "/path-to-npm-home/bin/npm"
-    ...
-}
-```
+### Configurable parameters
 
-## Quick Start
+**Exhort Snyk Token** :
+<br >The Snyk token allows Exhort to authenticate with the Snyk Vulnerability Database.
+If a Snyk token is not provided, Snyk vulnerability information is not displayed.
 
-Install the `Red Hat Dependency Analytics` extension from *Extensions Marketplace*.
+If you leave this field blank, the following informational message is displayed.
+<br >![ Screenshot of the empty token dialog box ](images/screenshots/empty-token.png)
 
-Configure the Snyk Token in the *extension workspace settings*. Please find configuration instructions [here](#configuration).
-On successfully entering the Snyk token, the Red Hat Dependency Analytics report can provide detailed information about security vulnerabilities unique to Snyk and vulnerabilities having publicly known exploits.
-<br >**IMPORTANT** Please note that if a valid Snyk Token is not provided in the extension workspace settings, Snyk vulnerabilities will not be displayed.
+If you enter a invalid Snyk token, a pop-up message alerts you that your Snyk token is not valid.
+![ Screenshot of the invalid token dialog box ](images/screenshots/invalid-token.png)
 
-Open or edit a manifest file *(pom.xml/package.json)* to trigger inline `Component Analysis`. This feature scans your application and provides inline feedback on any dependencies identified with security vulnerabilities.
+If you need a new Snyk token, you can generate a new token [here](https://app.snyk.io/login?utm_campaign=Code-Ready-Analytics-2020&utm_source=code_ready&code_ready=FF1B53D9-57BE-4613-96D7-1D06066C38C9).
```

Comment:
The link could be simplified to https://app.snyk.io/account

Reply:
See my previous comment above regarding this topic.